docs/src/api/class-cdpsession.md

```diff
@@ -64,6 +74,15 @@ Protocol method name.
 
 ### param: CDPSession.send.params
 * since: v1.8
+* langs: js, python
 - `params` ?<[Object]>
 
 Optional method parameters.
+
+### param: CDPSession.send.params
+* since: v1.8
+* langs: csharp
+  - alias-csharp: args
+- `params` ?<[Dictionary<string, Object>]>
+
+Optional method parameters.
```

Comment:
We might need to declare in this file "On" as well.

Reply:
Added AddEventListener and RemoveEventListener